psi4/src/psi4/fisapt/fisapt.cc

```diff
@@ -1125,7 +1256,7 @@ void FISAPT::dHF() {
     std::shared_ptr<Matrix> LH_B(T->clone());
     LH_B->copy(T);
     LH_B->add(V_B);
-    if (reference_->external_pot()) LH_B->add(matrices_["VE"]);
```

Comment:
These LE_A and LE_B energies (and some others) are not tested and they do not to contribute the SAPT0 energies printed at the end of the output files. Because of this, I will not edit this part.